umn/source/service_overview/ecs_specifications_and_types/large-memory_ecss.rst

```diff
@@ -16,39 +16,100 @@ Large-memory ECSs provide an even larger amount of memory than memory-optimized
+   +----------------+-------+--------------+------------------------------------+-------------+-----------------+--------------+---------------------+--------------------------------+
+
+.. table:: **Table 2** E3 ECS specifications
+
+   +----------------+-------+--------------+------------------------------------+----------------------+-----------------+--------------+---------------------+--------------------------------+
+   | Flavor         | vCPUs | Memory (GiB) | Maximum/Assured Bandwidth (Gbit/s) | Maximum PPS (10,000) | NIC Multi-Queue | Maximum NICs | Virtualization Type | Hardware                       |
+   +================+=======+==============+====================================+======================+=================+==============+=====================+================================+
+   | e3.7xlarge.12  | 28    | 348          | 25/12                              | 280                  | 8               | 8            | KVM                 | CPU: Intel® Xeon® Skylake 6151 |
+   +----------------+-------+--------------+------------------------------------+----------------------+-----------------+--------------+---------------------+--------------------------------+
+   | e3.14xlarge.12 | 56    | 696          | 25/25                              | 500                  | 16              | 8            | KVM                 |                                |
+   +----------------+-------+--------------+------------------------------------+----------------------+-----------------+--------------+---------------------+--------------------------------+
+   | e3.26xlarge.14 | 104   | 1466         | 30/20                              | 550                  | 16              | 8            | KVM                 | CPU: Intel® Xeon® Skylake 8176 |
+   +----------------+-------+--------------+------------------------------------+----------------------+-----------------+--------------+---------------------+--------------------------------+
+   | e3.52xlarge.14 | 208   | 2932         | 40/40                              | 1000                 | 32              | 8            | KVM                 |                                |
+   +----------------+-------+--------------+------------------------------------+----------------------+-----------------+--------------+---------------------+--------------------------------+
 
 Notes
 -----
 
 -  Large-memory ECSs do not support NIC hot swapping.
 
--  E3 ECSs support the following OSs that have been verified:
-
-   SUSE Enterprise Linux Server 12 SP2 64bit
-
-   SUSE Enterprise Linux Server 12 SP3 64bit
-
--  E3 ECSs can use ultra-high I/O EVS disks as the system disk and data disks.
-
--  The primary and extension NICs of a large-memory ECS have specified application scenarios. For details, see :ref:`Table 2 <en-us_topic_0038024694__table1642803151326>`.
+-  Specifications of large-memory ECSs cannot be modified.
```

Comment:
Please change it to:
Specifications of large-memory ECSs can be modified only within the same flavor family